### PR TITLE
feature- Product Card component

### PR DESCRIPTION
**Pull request by messkalina**
Add Product Card component for the product items ( stays/ experiences)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -40,7 +40,15 @@ function App() {
         rating={4.85}
         yearsHosting={1}
       />
+       <ProductCard 
+        image="https://a0.muscache.com/im/pictures/hosting/Hosting-U3RheVN1cHBseUxpc3Rpbmc6MTEzMTA4OTQ5ODA0MDcwMTE4Mw%3D%3D/original/4f7a276e-9995-4b32-bda6-300de0619b25.png?im_w=2560&im_q=highq"
```
Comment (orYoffe) [inline src/App.jsx:44]: we shouldn't use the images from other websites.. they might change, we should add images ourselves to our repo and link to them

File: src/App.jsx [hunk 2]
```diff
@@ -40,7 +40,15 @@ function App() {
         rating={4.85}
         yearsHosting={1}
       />
+       <ProductCard 
+        image="https://a0.muscache.com/im/pictures/hosting/Hosting-U3RheVN1cHBseUxpc3Rpbmc6MTEzMTA4OTQ5ODA0MDcwMTE4Mw%3D%3D/original/4f7a276e-9995-4b32-bda6-300de0619b25.png?im_w=2560&im_q=highq" 
+        title="Cozy Apartment in City Center" 
+        host="Hosted by Wendy and Elisa"
+        price="Coming soon"
```
Comment (orYoffe) [inline src/App.jsx:47]: can we pass these values as props so we can reuse this component?

**Review by orYoffe — COMMENTED** [review]
Screenshot 2024-09-24 at 18 30 12
seems the share button isn't rendering, does it happen on your machine too?

**Comment by orYoffe** [comment]
> Screenshot 2024-09-24 at 18 30 12 seems the share button isn't rendering, does it happen on your machine too?

also the text is not aligned to the left like in the design in the ticket https://github.com/orgs/ReDI-School/projects/2/views/1?pane=issue&itemId=80422309

File: src/App.jsx (hunk 1 above)
Comment (messkalina) [reply src/App.jsx]: Image linked to asset in repo

**Comment by messkalina** [comment]
> > Screenshot 2024-09-24 at 18 30 12 seems the share button isn't rendering, does it happen on your machine too?
> 
> also the text is not aligned to the left like in the design in the ticket https://github.com/orgs/ReDI-School/projects/2/views/1?pane=issue&itemId=80422309

Those issues has been fixed, thank you!

**Comment by dvnishshanka** [comment]
image

Hi, When I try to run it on my local machine the share icon still doesn't show to me.

**Comment by messkalina** [comment]
Screenshot 2024-09-26 at 18 59 13
 
I created this component but couldnt resolve conflict yet to merge it

**Review by GabrielMelhem — APPROVED** [review]

**Pull request merged**